source/preparing/jdk.rst

```diff
@@ -21,4 +21,6 @@ Oracle provides free downloads of the Java Development Kit on their website. Ens
 
 * `Java Development Kit 8 <https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html>`__
 
+If you don't want to create an Oracle account in order to download, you can also download the JDK from `AdoptOpenJDK <https://adoptopenjdk.net/?variant=openjdk8&jvmVariant=hotspot>`__.
```

Comment:
I think it's a bit misleading as-is. The OpenJDK is not the same beast as the Oracle JDK, as far as I know, and I'm not sure that's the right way to say "an Oracle account is required to download Oracle's JDK". Perhaps instead: 
"An open-source alternative is the OpenJDK, which is available from (yadda yadda)"